include/arch/arm/arch/64/mode/machine/registerset.h

```diff
@@ -274,6 +274,7 @@ struct user_context {
     user_breakpoint_state_t breakpointState;
 #endif /* ARM_BASE_CP14_SAVE_AND_RESTORE */
 #ifdef CONFIG_HAVE_FPU
+    word_t _padding;
```

Comment:
The comment on line 263 makes no sense to me. Is it trying to say how big this structure is? If so it appears completely wrong by a few multiples. (I looked because this padding is changing the struct size)

Reply:
That's copy & pasted from aarch32. I'll remove it, just like I did for the tcb.